src/Commands/BannedKeywordsBag.ts

```diff
@@ -0,0 +1,204 @@
+            "implements": true,
+            "implicit": true,
+            "import": true,
+            "in": true,
+            "input": true,
+            "instanceof": true,
+            "interaface": true,
+            "interface": true,
+            "internal": true,
+            "is": true,
+            "lambda": true,
+            "let": true,
+            "lock": true,
+            "log": true,
+            "log10": true,
+            "long": true,
+            "module": true,
+            "namespace": true,
+            "native": true,
+            "new": true,
+            "next": true,
+            "nil": true,
+            "not": true,
+            "null": true,
+            "number": true,
+            "object": true,
+            "of": true,
+            "open": true,
+            "operator": true,
+            "or": true,
+            "out": true,
+            "override": true,
+            "package": true,
+            "params": true,
+            "partial": true,
+            "pass": true,
+            "pi": true,
+            "print": true,
+            "private": true,
+            "protected": true,
+            "public": true,
+            "raise": true,
+            "range": true,
+            "readonly": true,
+            "redo": true,
+            "ref": true,
+            "remove": true,
+            "require": true,
+            "rescue": true,
+            "retry": true,
+            "return": true,
+            "sbyte": true,
+            "sealed": true,
+            "self": true,
+            "set": true,
+            "short": true,
+            "sin": true,
+            "sizeof": true,
+            "sqrt": true,
+            "stackalloc": true,
+            "static": true,
+            "strictfp": true,
+            "string": true,
+            "struct": true,
+            "super": true,
+            "switch": true,
+            "symbol": true,
+            "synchronized": true,
+            "tan": true,
+            "then": true,
+            "this": true,
+            "throw": true,
+            "throwas": true,
+            "throws": true,
+            "transient": true,
+            "true": true,
+            "try": true,
+            "type": true,
+            "typeof": true,
+            "uint": true,
+            "ulong": true,
+            "unchecked": true,
+            "undef": true,
+            "unless": true,
+            "unsafe": true,
+            "until": true,
+            "ushort": true,
+            "using": true,
+            "value": true,
+            "var": true,
+            "virtual": true,
+            "void": true,
+            "volatile": true,
+            "when": true,
+            "where": true,
+            "while": true,
+            "with": true,
+            "write": true,
+            "yield": true,
+            "zeros": true,
+        };
+    }
+
+    /**
+     * Returns whether the variable name is allowed.
+     * 
+     * @param name  The variable name.
```

Comment:
Nit: for the `param` and general doc, it would be more accurate to say "a" instead of "the" IMO.